### PR TITLE
DRA-855 - facets moved to top

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Moved facets to the top
Did some refactoring on the SearchOverhead.vue
Added a few translations
Added grid option for showing results

### CODE REVIEW

File: src/components/search/SearchOverhead.vue [hunk 1]
```diff
@@ -23,14 +23,51 @@
...
+								}}
+							</div>
+						</Transition>
+						<button
+							:class="
+								searchResultStore.loading
+									? 'display-option list loading'
+									: searchResultStore.resultGrid
+									  ? 'display-option list'
+									  : 'display-option list active'
```
Comment (jorntx) [inline src/components/search/SearchOverhead.vue:51]: ouchhh - skating on thin ice here.... :)

File: src/components/search/SearchOverhead.vue [hunk 2]
```diff
@@ -23,14 +23,51 @@
...
-					</Transition>
+							<span class="material-icons">toc</span>
+						</button>
+						<button
+							:class="
+								searchResultStore.loading
+									? 'display-option grid loading'
+									: searchResultStore.resultGrid
+									  ? 'display-option grid active'
+									  : 'display-option grid'
```
Comment (jorntx) [inline src/components/search/SearchOverhead.vue:63]: ouchhh - skating on thin ice here.... :)

File: src/components/search/SearchResults.vue [hunk 3]
```diff
@@ -43,7 +52,6 @@ export default defineComponent({
 		const { t, locale } = useI18n();
 		const currentResults = ref([] as GenericSearchResultType[]);
 		const resultNr = ref(0);
```
Comment (jorntx) [inline src/components/search/SearchResults.vue:54]: Maybe rename to `hitsToShow`?

File: src/components/search/SearchResults.vue [hunk 4]
```diff
@@ -91,7 +84,16 @@ export default defineComponent({
...
+			watch(
+				() => searchResultStore.resultGrid,
+				(newBool: boolean) => {
+					if (newBool) {
+						resultNr.value = 40;
+					} else {
+						resultNr.value = 10;
 					}
 				},
 			);
```
Comment (jorntx) [inline src/components/search/SearchResults.vue:99]: Maybe refactor to (if `resultNr` is renamed to` hitsToShow`):

```
watch(
  () => searchResultStore.resultGrid,
  (isGrid: boolean) => {
    hitsToShow.value = isGrid ? 40 : 10;
  }
);
```

**Review by jorntx — COMMENTED** [review]
Looks good and works.

Only small suggestions for revisions.

File: src/components/search/SearchOverhead.vue (hunk 1 above)
Comment (jesperlauridsen) [reply src/components/search/SearchOverhead.vue:51]: Hear me out here! 2 ternary operators are okey - from there it gets too messy! Deal? :pray:

File: src/components/search/SearchResults.vue (hunk 3 above)
Comment (jesperlauridsen) [reply src/components/search/SearchResults.vue:54]: Good idea!

File: src/components/search/SearchResults.vue (hunk 4 above)
Comment (jesperlauridsen) [reply src/components/search/SearchResults.vue:99]: Much cleaner - nice.

**Review by jorntx — APPROVED** [review]
Looks good and tested out well

**Pull request merged**